app/dashboard/views.py

```diff
@@ -193,6 +194,63 @@ def remove_interest(request, bounty_id):
 
     return JsonResponse({'success': True})
 
+@require_POST
+@csrf_exempt
+def uninterested(request, bounty_id, profile_id):
+    """Remove party from given bounty
+
+    :request method: GET
+
+    Args:
+        bounty_id (int): ID of the Bounty
+        profile_id (int): ID of the interested profile
+
+    Returns:
+        dict: The success key with a boolean value and accompanying error.
+    """
+
+    session_profile_id = request.session.get('profile_id')
+    if not session_profile_id:
+        return JsonResponse(
+            {'error': 'You must be authenticated!'},
+            status=401)
+
+    try:
+        bounty = Bounty.objects.get(pk=bounty_id)
+    except Bounty.DoesNotExist:
+        return JsonResponse({'errors': ['Bounty doesn\'t exist!']},
+                            status=401)
+
+
+    if not bounty.is_funder(request.session.get('handle').lower()):
```

Comment:
we already have this check in bounty model, so I will use it rather than defining it once again